mlflow/models/model.py

```diff
@@ -792,6 +792,11 @@ def load(cls, path) -> "Model":
             # Load the Model object from a remote model directory
             model2 = Model.load("s3://mybucket/path/to/my/model")
         """
+        if path is None:
+            raise MlflowException.invalid_parameter_value(
+                "Unable to load model, please ensure that you've "
+                "specified the correct path to the model."
+            )
```

Comment:
can we raise this in `download_artifacts`?

Reply:
But what if users just want to download artifacts? e.g. run artifacts, and there's no model, then we can't raise in download_artifacts

Reply:
Emm we can raise an exception like `Unable to download artifacts from path ...`, but this message seems confusing to users too if they're just using `load_model`?